cmd/launcher/options.go

```diff
@@ -83,6 +83,7 @@ func parseOptions(subcommandName string, args []string) (*launcher.Options, erro
 		flCompactDbMaxTx         = flagset.Int64("compactdb-max-tx", 65536, "Maximum transaction size used when compacting the internal DB")
 		flConfigFilePath         = flagset.String("config", defaultConfigFilePath, "config file to parse options from (optional)")
 		flExportTraces           = flagset.Bool("export_traces", false, "Whether to export traces")
+		flIngestServerURL        = flagset.String("ingest_url", "localhost:4317", "Where to export traces and logs")
```

Comment:
Naive question -- with this default, does it default to sending logs there? It feels like we should default to `""`

Reply:
It won't, just because the default for `export_traces` is false. But I can make this an empty string!